src/Microsoft.ML.Data/Scorers/FeatureContributionCalculatingTransformer.cs

```diff
@@ -493,4 +575,71 @@ public void GetMetadata<TValue>(string kind, int col, ref TValue value)
             }
         }
     }
+
+    // TODO DOcumentation
+    public sealed class FeatureContributionCalculatingEstimator : TrivialEstimator<FeatureContributionCalculatingTransformer>
+    {
+        private readonly string _featureColumn;
+        private readonly IPredictor _predictor;
+        private readonly bool _stringify;
+
+        public static class Defaults
+        {
+            public const int Top = 10;
+            public const int Bottom = 10;
+            public const bool Normalize = true;
+            public const bool Stringify = false;
+        }
+
+        // TODO Documentation
+        public FeatureContributionCalculatingEstimator(IHostEnvironment env, IPredictor predictor, string featureColumn,
+            int top = Defaults.Top,
+            int bottom = Defaults.Bottom,
+            bool normalize = Defaults.Normalize,
+            bool stringify = Defaults.Stringify)
+            : base(Contracts.CheckRef(env, nameof(env)).Register(nameof(FeatureContributionCalculatingTransformer)),
+                  new FeatureContributionCalculatingTransformer(env, predictor, featureColumn, top, bottom, normalize, stringify))
+        {
+            _featureColumn = featureColumn;
+            _predictor = predictor;
+            _stringify = stringify;
+        }
+
+        public override SchemaShape GetOutputSchema(SchemaShape inputSchema)
+        {
+            Host.CheckValue(inputSchema, nameof(inputSchema));
+            var result = inputSchema.ToDictionary(x => x.Name);
+
+            // Add Score column.
+            var scoreMetadata = new List<SchemaShape.Column>();
```

Comment:
I think this is true for regression only I will need to add the score columns for each predictionkind (regression, binaryclassification, multiclassclassification, ranking)